tests/test_droid_hook_session_mapping.py

```
@@ -0,0 +1,245 @@
+    return None
+
+
+def fail(message: str) -> int:
+    print(f"FAIL: {message}")
+    return 1
+
+
+def main() -> int:
+    try:
+        cli_path = resolve_cmux_cli()
+    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)